src/cli/cli.cc

```diff
@@ -3627,6 +3700,86 @@ int main (const int argc, const char* argv[]) {
       exit(exitCode);
     }
   });
+  
+  createSubcommand("setup", { "--platform", "--yes" }, false, [&](const std::span<const char *>& options) -> void {
+    auto win = platform.win;
+
+    auto help = false;
+    String targetPlatform;
+    auto targetAndroid = false;
+    auto targetWindows = false;
+    auto yes = false;
+    String yesArg;
+    for (auto const arg : options) {
+      if (is(arg, "-h") || is(arg, "--help")) {
+        help = true;
+      }
+
+      auto platform = optionValue("setup", arg, "--platform");
+      if (platform.size() > 0) {
+        targetPlatform = platform;
+      }
+
+      if (is(arg, "-y") || is(arg, "--yes")) {
+        yes = true;
+      }
+
+      if (help) {
+        break;
+      }
+    }
+
+    // Note that multiple --platforms aren't supported by createSubcommand()
+    if (is(targetPlatform, "android")) {
+      targetAndroid = true;
+    } else if (is(targetPlatform, "windows")) {
+      targetWindows = true;
+    } else {
+      help = true;
+    }
+
+    if (help) {
+      printHelp("setup");
+      exit(1);
```

Comment:
should be `exit(0)`

Reply:
My thinking here is that if a `ssc setup` is scripted with the wrong arguments, we want to return an exit code indicating failure.

Reply:
ah, ok, that makes sense